command/agent_info.go

```diff
@@ -53,14 +54,26 @@ func (c *AgentInfoCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Sort and output agent info
 	var stats map[string]interface{}
 	stats, _ = info["stats"]
+	stats_keys := make([]string, 0, len(stats))
```

Comment:
Keep in mind that stats_keys is not a idiomatic variable notation in Go. It should be statsKeys (camelcase). My 2 cents.